### DISCUSSION TITLE
Confused on `base` and `site` values for Github pages

### DISCUSSION THREAD

**Discussion by C-Loftus** [withastro/starlight#2158] [Q&A] [answered]
Hello, I am working on deploying a starlight site with astro

I have a github action that works to deploy my site but whichever way I try to deploy it, either the links are broken or the css is missing.

The config I have is here: https://github.com/C-Loftus/talon-ai-tools/blob/main/docs/astro.config.mjs

I have the domain `colton.place` which is mapped to github pages. So the namespace for the pages site for the particular repo would be `colton.place/talon-ai-tools`

As I have it now, it does not render any css https://colton.place/talon-ai-tools/ However, if I change the site to just be colton.place and add the base as talon-ai-tools then it breaks the sidebar links since they end up going to the root of the domain at `/`

Is there a way I can fix this? I have looked in previous discussions and can not find anything on the docs.

Thank you

**Comment by C-Loftus** [comment, marked as answer]
Fixed this with the following.

```
    site: "https://colton.place/talon-ai-tools",
    base: "talon-ai-tools",
```

**Reply by delucis** [reply]
Thanks for sharing @C-Loftus!

I wonder if this might be a bug somewhere in our logic 🤔 In theory, the following should be sufficient I think:

```js
    site: "https://colton.place/",
    base: "talon-ai-tools",
```

But maybe we’re not correctly using `base` somewhere.
